src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,59 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(String type, String fruit, int quantity) {
+        switch (type) {
+            case "b":
+                operation = Operation.BALANCE;
+                break;
+            case "s":
+                operation = Operation.SUPPLY;
+                break;
+            case "p":
+                operation = Operation.PURCHASE;
+                break;
+            case "r":
+                operation = Operation.RETURN;
+                break;
+            default:
+        }
```

Comment:
i don't know can we create logic in constructor?

Reply:
I create FruitTransaction in constructor, it looks not nice but I don't know how to simplyfy it. In a sample of this class was such line BALANCE("b") but I dont know what this parentheses mean. Probably it has som conection with way how to simplify this.